docs/manual.md

```diff
@@ -263,6 +292,8 @@ sel4cp does not provides timers, nor any *sleep* API.
 After initialisation, activity in the system is initiated by an interrupt causing a `notified` entry point to be invoked.
 That notified function may in turn notify or call other protection domains that cause other system activity, but eventually all activity indirectly initiated from that interrupt will complete, at which point the system is inactive again until another interrupt occurs.
 
+An interrupt must be acknowledged with `sel4cp_irq_ack()` in the `notified()` function, otherwise it will not be fired again.
```

Comment:
An interrupt doesn't necessarily have to be acknowledged in the `notified()` function. My understanding is that the interrupt will continue to be fired, it just won't be delivered to seL4. Perhaps it would be better to say "otherwise the interrupt will not be delivered again".

Reply:
Good point, I updated the wording.